modules/calypsoify/_colors.scss

```diff
@@ -1,74 +1,9 @@
 @import 'muriel-wpcom';
-// Colors that aren't pulled from Calypso or Core:
-
-$_0099d8: #0095d6; // Pacific Blue
-$_0096c3: #0094c1; // Pacific Blue
-$_008ab3: #0088b2; // Bondi Blue
-$_004967: #004966; // Prussian Blue
-
-// These colors are used several times in Calypso, but never actually stored in a variable.
-$_d9e3ea: #d7e1e9; // Pattens Blue
-$rgba-grey: rgba(200,215,225,0.5);
-
-$color-primary: $muriel-blue-500;
-$color-primary-light: $muriel-blue-300;
-$color-primary-dark: $muriel-blue-700;
-
-$color-accent: $muriel-hot-pink-500;
-$color-accent-light: $muriel-hot-pink-300;
-$color-accent-dark: $muriel-hot-pink-700;
-
-$color-link: $muriel-blue-500;
-$color-link-dark: $muriel-blue-700;
-
-$color-error: $muriel-hot-red-500;
-$color-error-light: $muriel-hot-red-300;
-$color-error-dark: $muriel-hot-red-700;
+@import '../../node_modules/@automattic/calypso-color-schemes/dist/calypso-color-schemes.css';
```

Comment:
Heads up (from #12277):

https://github.com/Automattic/jetpack/blob/b9728ae96046aef852a3161c3475306f5e1977a2/_inc/client/scss/style.scss#L15

Reply:
Can you elaborate on what that means for calypsoify's styling? I don't see a direct relation as I'm importing css custom props whereas that file you mentioned only contains sass color variables.

Reply:
I was just making sure that'd you'd seen the change to that line. There may not be any issues.

Reply:
thanks for the heads up 🙂